scripts/provisioning/s3_setup

```diff
@@ -88,7 +89,10 @@ def main():
     if args.command == 'post_install':
       from postinstallcmd import PostInstallCmd
       logger.info('post_install started.')
-      PostInstallCmd(args.config, args.services).process()
+      with open('/etc/cortx/s3/post_install.lock', 'w' as lock:
+        fcntl.flock(lock, fcntl.LOCK_EX)
+        PostInstallCmd(args.config, args.services).process()
```

Comment:
id there chance PostInstallCmd does system exit, how lock is released

Reply:
Add logs